tests/src/Functional/NewsIntegrationTest.php

```diff
@@ -73,24 +76,33 @@ public function testCreateNews() {
 
     // Create a News item.
     $this->drupalGet('node/add/oe_sc_news');
-    $page = $this->getSession()->getPage();
     $page->fillField('Title', 'Example title');
     $page->fillField('Content', 'Example Content');
     $page->fillField('Introduction', 'Example Introduction');
-    $page->fillField('Date', '2022-01-24');
     $media_name = $media_image->getName() . ' (' . $media_image->id() . ')';
     $page->fillField('Media item', $media_name);
     $page->fillField('Caption', 'Starter Image caption');
     $page->pressButton('Save');
 
-    // Assert media document has been created.
+    // Assert media document has been created and publication date
+    // is current date because it's not fulfilled.
     $assert_session->pageTextContains('News Example title has been created.');
     $assert_session->pageTextContains('Example title');
     $assert_session->pageTextContains('Example Content');
     $assert_session->pageTextContains('Example Introduction');
```

Comment:
We can insert the comment about pub date here - but depends on the solution we choose for testing it.
```suggestion
    $assert_session->pageTextContains('Example Introduction');
    // Assert that publication date was filled with a default value.
```